spec/requests/api/v1/users/create_spec.rb

```diff
@@ -0,0 +1,122 @@
+require 'rails_helper'
+
+describe 'POST api/v1/users/', type: :request do
+  let(:user)            { User.last }
+  let(:failed_response) { 422 }
+  let(:internal_server_error) { 500 }
+
+  describe 'POST create' do
+    let(:email)                 { 'test@test.com' }
+    let(:password)              { '12345678' }
+    let(:password_confirmation) { '12345678' }
+    let(:name)                  { 'Sebastian' }
+    let(:gender)                { 'male' }
+
+    let(:params) do
+      {
+        user: {
+          email: email,
+          password: password,
+          password_confirmation: password_confirmation,
+          name: name,
+          gender: gender
+        }
+      }
+    end
+
+    it 'returns a succesful response' do
+      post user_registration_path, params: params, as: :json
+
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'creates the user' do
+      expect do
+        post user_registration_path, params: params, as: :json
+      end.to change(User, :count).by(1)
+    end
+
+    it 'returns the user' do
+      post user_registration_path, params: params, as: :json
+
+      expect(json[:user][:id]).to eq(user.id)
```

Comment:
You can use https://github.com/waterlink/rspec-json_expectations to check the JSON response.